packages/ui/src/components/organisms/Form/Validator/_stories/components/Story/Story.tsx

```
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+import { IValidationSchema, registerValidator, ValidatorProvider } from '../../../../Validator';
+import { JSONEditorComponent } from '../../../../Validator/_stories/components/JsonEditor/JsonEditor';
+import { ValidatorParams } from '../../../../Validator/_stories/components/ValidatorParams';
+import { initialContext } from './context';
+import { ErrorsList } from './ErrorsList';
+import { initialSchema } from './schema';
+
+const evenNumbersValidator = (value: number) => {
+  // Ignoring validation if value is not a number
+  if (isNaN(value)) return true;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)